src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/faktavurderinger/Vurdering.kt

```diff
@@ -167,7 +167,7 @@ data class VurderingAldersVilkår(
     }
 }
 
-data class VurderingMottarSykepengerForFulltidsstilling private constructor(
+data class VurderingMottarSykepengerForFulltidsstilling constructor(
```

Comment:
Nå er denne konstruktøren public kun for testbarhet? Kan vi unngå det på noen måte?

Reply:
Jeg kom ikke på noen annen måte, syntes det var litt dumt. 
Trenger det for å kunne teste å lagre ned med gammel mangler data, men den kaster feil om jeg prøver å bruke den andre konstruktøren 😢 
Hvis noen har forslag så endrer jeg gjerne!